build

```diff
@@ -3,14 +3,29 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+LINK_OPERATOR="="
+
+: ${ARCH_LIST:="amd64 arm64"}
```

Comment:
@heyitsanthony Your point is well taken. I would not say many, but some do.
I think the ability to build for all arches is a valuable one though, to be used before commits/PRs say.  Maybe build only for the host arch as default with an option to build ARCH_LIST?